code/modules/antagonists/traitor/objectives/sabotage_machinery.dm

```diff
@@ -0,0 +1,133 @@
+/datum/traitor_objective_category/sabotage_machinery
+	name = "Sabotage Worksite"
+	objectives = list(
+		/datum/traitor_objective/sabotage_machinery = 1,
+	)
+
+/datum/traitor_objective/sabotage_machinery
```

Comment:
For this objective, I think it would be interesting if you let them destroy it normally but at a reduction in rewards (both in TC and progression)
I prefer traitor objectives to be more open-ended.

Reply:
My problem with this is that there's no method I can find of allowing open-ended destruction which doesn't simply include clicking on the machine once with a screwdriver and then a crowbar, which causes the machine to cease to exist as far as code is concerned.
If we'd prefer to leave it open-ended then I completely understand the reasoning but it'd probably be best to just close this PR as a failed experiment, as preventing that behaviour was sort of the sole purpose.

Reply:
I guess the alternative is to make the objective explicit about destroying the machine board rather than the machine?
As other people have pointed out, there are currently only two machine boards for lathes in existence on any map and no means of procuring more.
This is either a positive (you should simply protect it better) or a negative (well now I can't print any department-specific gear), although I could also simply add a means of acquiring more in the same PR.

Reply:
That's fine though, clicking with screwdriver and then crowbar is fine. Just make it give reduced TC and progression, players can choose the lazy way out or they can make extra TC/progression, it's up to them.

Reply:
My sort of goal here is to reduce or remove instances where completing an objective has literally no impact at all on anyone.
When you deconstruct a machine with tools you can simply immediately reconstruct it, meaning that you did not inconvenience anyone at all and as essentially no crime happened nobody has any reason to investigate either, unless you are caught in the act (a span of about 15 seconds) nobody will ever know.

I can add it back in with reduced TC and then also try to make it so that non-roundstart protolathes need to be manually linked to the vault in the same manner as the ORM so that deconstructing it still leaves some external task to complete before the machine is functional?
That would address both of our issues, but might constitute a different PR.

Reply:
It does *do* something, which is unlink from the ore silo.

Reply:
> It does _do_ something, which is unlink from the ore silo.

You know for some reason I was CONVINCED that only the ORM did this but you're right it does apply to all three. 
In that case yeah I'll just put the signal back as a reduced reward.